kroma-validator/l2_output_submitter.go

```diff
@@ -181,6 +287,16 @@ func (l *L2OutputSubmitter) repeatSubmitL2Output(ctx context.Context) {
 // If it needs to wait, it will calculate how long the validator should wait and
 // try again after the delay.
 func (l *L2OutputSubmitter) trySubmitL2Output(ctx context.Context) (time.Duration, error) {
+	if l.isValManagerEnabled {
+		hasSubmittedStartTx, err := l.tryStartValidator(ctx)
+		if err != nil {
+			return l.cfg.OutputSubmitterRetryInterval, err
+		}
+		if !hasSubmittedStartTx {
+			return l.cfg.OutputSubmitterRetryInterval, nil
+		}
```

Comment:
How about move this logic to `CanSubmitOutput` function? Before node sync completed, validator client cannot check its status and try to start validator.